### ISSUE TITLE
Too difficult to reimplement your project !

### ISSUE THREAD

**Issue by wqz960** [MarekKowalski/DeepAlignmentNetwork#50]
Hi! I am very interesting in your project! But it seems that there are some difference between the Traindataset process script and the actual script in your paper. And can you provide me the training detail, such as the epoch for stage1 and stage2, and the learning rate. The network you provide on the Internet also cant reach the accuracy in your paper! Looking forward to you kind rely. Thank you!

**Comment by MarekKowalski** [comment]
Hi,

The scripts for creating the datasets are the same as in the paper, the pre-trained DNNs should give results exactly the same as reported in the paper. Which DNNs are you using, and which results do not reproduce?

As for the epochs: I do not have the exact numbers, but I did stop the training of stage1 once it stopped improving quickly, I did not train it to the very end.

Best regards,

Marek

**Comment by wqz960** [comment]
@MarekKowalski Hi！Did you test the 300Wtestset using your network on the Internet? If I didn't make a stupid mistake, There is a gap between your result in paper and your result network on the Internet, the network on the Internet seems 0.034+, But your paper result is 0.0319. Thank you!

**Comment by MarekKowalski** [comment]
Hi, like I said before, the results should be identical to the ones in the paper. Which DNN did you use and what results do you get on the other datasets?

Thanks,

Marek

**Comment by wqz960** [comment]
Hi! @MarekKowalski  Thank you for your help! there is another question, the bounding box in your pkl file, are the boxes produced by your own face detector, or the bounding boxes are from the 300W web, you just changed the form from mat to pkl?

**Comment by MarekKowalski** [comment]
Hi,

The bounding boxes are from the 300W dataset.

Marek
